docs/synthetic-tokens/what-are-synthetic-assets.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Synthetic tokens
+sidebar_label: Synthetic tokens
```

Comment:
```suggestion
sidebar_label: Synthetic Tokens
```

Reply:
Think it is strange to have one page titled as "Synthetic Tokens" and another as "Priceless Synthetic Tokens". In my opinion, the content from these two pages could be combined into one page